emva1288/process/plotting.py

```diff
@@ -131,20 +131,20 @@ def reduce_ticks(self, ax, axis, n=4):
 
 
 class PlotSensitivity(Emva1288Plot):
-    name = 'Sensitivity'
-    xlabel = '$\mu_p$ [mean number of photons/pixel]'
-    ylabel = '$\mu_y - \mu_{y.dark}$ [DN]'
+    name = r'Sensitivity'
```

Comment:
Converting this to a raw string, along with all other strings that do not contain backslashes, is unnecessary.

Reply:
Personally, I think only using them when necessary makes it clearer _why_ they are necessary. Applying this everywhere, it seems more likely that we'll end up with an untreated `\n`.

@fariza Any thoughts on this?

Reply:
Sounds like a strong argument to me. :-)